aspnet/getting-started.rst

```diff
@@ -0,0 +1,53 @@
+Getting Started
+===============
+
+1. Install `.NET Core`_
+
+2. Create a new .NET Core project:
+
+  .. code-block:: console
+    
+    mkdir aspnetcoreapp
+    cd aspnetcoreapp
+    dotnet new
+
+3. Update your ``project.json`` file to add the package for the Kestrel HTTP server and the ASP.NET Core hosting model:
+
+  .. literalinclude:: getting-started/sample/aspnetcoreapp/project.json
+    :language: c#
+    :emphasize-lines: 11
+
+4. Restore the packages for your project:
+
+  .. code-block:: console
+    
+    dotnet restore
+
+5. Add a ``Startup.cs`` file that defines the request handling logic your app:
+
+  .. literalinclude:: getting-started/sample/aspnetcoreapp/Startup.cs
+    :language: c#
+
+6. Replace the code in the static ``Main`` method in ``Program.cs`` to setup and start the Web host:
+
+  .. literalinclude:: getting-started/sample/aspnetcoreapp/Program.cs
+    :language: c#
+    :emphasize-lines: 2,10-15
+
+7. Build and run the app:
```

Comment:
Probably leave as is - but consider:
Run the app (the ``dotnet run`` command will build the app when it's out of date):